src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -918,7 +1006,7 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                             /> Apply Log Scale
                         </label></div>
                     )}
-                    <div className="form-group" style={{opacity:(axisSelection.dataType === CLIN_ATTR_DATA_TYPE ? 0 : 1)}}>
+                    {(axisSelection.dataType !== CLIN_ATTR_DATA_TYPE && axisSelection.dataType !== GENESET_DATA_TYPE) && (<div className="form-group">
```

Comment:
I was using `opacity` because that way the size of the panel doesn't change, which may be smoother UX. @alisman what do you think?

Reply:
If you prefer, I can keep the "gene" selection box hidden when selecting the clinical attribute instead of removing it to keep the size of the panel, as you implemented. Be aware though that now there are have four selection boxes (gene and gene set boxes are independent).

Reply:
@adamabeshouse I agree we want to avoid jumpiness.   Oleg, I'm not sure how to ansewr question.  I think I need to see it to undersatnd.  Can you put screenshot or do demo for me?

Reply:
@oplantalech would there ever be the gene and geneset box visible at the same time?

Reply:
@adamabeshouse No. Geneset box only appears when the geneset datatype is selected

Reply:
@oplantalech would it be possible to reuse the same component, and just insert different options and a different label and a different onChange method?

Reply:
@adamabeshouse I guess so, but I think it is more logical to be a separate component. If you want to keep the same size for the selection boxes, I can keep the gene box in place without the option to see it when clinical attributes are selected (what you implemented).